src/main/java/com/example/simpleannotation/model/BuilderGeneratedClass.java

```diff
@@ -0,0 +1,181 @@
+package com.example.simpleannotation.model;
+
+import com.example.simpleannotation.exceptions.ClassGenerationException;
+import com.example.simpleannotation.model.descriptors.AttributeDescriptor;
+import com.example.simpleannotation.model.descriptors.ConstructorDescriptor;
+import com.example.simpleannotation.model.descriptors.MethodDescriptor;
+
+import java.util.*;
+import java.util.stream.Collectors;
+
+public final class BuilderGeneratedClass extends GeneratedClass {
+
+    private static final String BUILDER_NAME = "%sBuilder";
+    private static final String INSTANCE = "instance";
+    private static final String PUBLIC = "public";
+    private static final String FINAL = "final";
+    private static final String PRIVATE = "private";
+
+    private final BuilderAnnotatedClass annotatedClass;
+
+    public BuilderGeneratedClass(BuilderAnnotatedClass annotatedClass) {
+        this.annotatedClass = annotatedClass;
+        this.setClassName(computeBuilderName());
+        this.setPackageName(annotatedClass.getClassToBuildPackageName());
+        if (annotatedClass.useSingletonBuilder()) {
+            addSingletonPattern();
+        } else {
+            addSimpleConstructor();
+        }
+        addCommonFields();
+        if (annotatedClass.useFluentBuilder()) {
+            addFluentSetters();
+        } else {
+            addSimpleSetters();
+        }
+        addBuildMethod();
+    }
+
+    private void addCommonFields () {
```

Comment:
delete extra space     private void addCommonFields() {

Reply:
Done. Revised in [pull request 8](https://github.com/OscarCamacho/simpleannotation/pull/8)